src/firebase/firebase_config.js

```diff
@@ -25,4 +42,5 @@ const firebaseConfig = {
 const app = firebase.initializeApp(firebaseConfig);
 const db = getFirestore(app);
 
+export const auth = app.auth()
```

Comment:
nit. Can we define auth, and then include it into the export statement at the bottom? I.e.

const auth = app.auth()
export {app, db, auth}

Reply:
Absolutely, I'll make that change.